force_bdss/events/tests/test_base_driver_event.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+
+from traits.api import Instance, Int
+
+from force_bdss.core.data_value import DataValue
+from force_bdss.events.base_driver_event import BaseDriverEvent
+
+
+class DummyEvent(BaseDriverEvent):
+    stateless_data = Int(1)
+    stateful_data = Instance(DataValue)
+
+
+class TestBaseDriverEvent(unittest.TestCase):
+
+    def test_getstate_base_event(self):
+        event = BaseDriverEvent()
+        self.assertFalse(event.__getstate__())
+
+        event = DummyEvent(stateful_data=DataValue(value=1))
+        self.assertDictEqual(
```

Comment:
FYI the `__getstate__` format will change after the (de)serialization PR is merged. The event is now serialized to the same format as the `BaseModel`: `{"id": id, "model_data": data}`.